code/solutions/language-extensions/language-extensions-part2.yaml

```diff
@@ -0,0 +1,58 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - Language extensions (uksb-1q9p31idr).
+
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks:
+        - W3011
+
+Parameters:
+  DeletionPolicyParameter:
+    Type: String
+    AllowedValues: [Delete, Retain]
+    Default: Delete
+
+  LatestAmiId:
+    Description: The ID of the region-specific Amazon Machine Image to use.
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
+
+Transform: AWS::LanguageExtensions
+
+Resources:
+  Dashboard:
+    Type: AWS::CloudWatch::Dashboard
+    Properties:
+      DashboardBody:
+        Fn::ToJsonString:
+          start: -PT6H
+          periodOverride: inherit
+          widgets:
+            - type: metric
+              x: 0
+              "y": 7
```

Comment:
@pranjal-gururani should this not have the quotes, for consistency?

Reply:
Actually - my bad, the formatter we use would have added the quotes, and I believe this is expected -and makes sense- to distinguish a string -like in your case- from a value that might be interpreted as a boolean.  I think you can leave it as it is or, for consistency, you could try adding quotes in other parts as well in the widget (unless the formatter expunges the quotes, deeming them not needed).  Let me know what you think!

Reply:
I am going to leave "y" quoted, as the formatter will add the other quotes.